app/src/main/java/org/sopt/android_hyorim_30th/ui/signin/SignInViewModel.kt

```diff
@@ -1,11 +1,36 @@
 package org.sopt.android_hyorim_30th.ui.signin
 
+import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import dagger.hilt.android.lifecycle.HiltViewModel
+import org.sopt.android_hyorim_30th.data.local.SOPTSharedPreference
+import javax.inject.Inject
 
-class SignInViewModel : ViewModel() {
+@HiltViewModel
+class SignInViewModel @Inject constructor(
+    private val preference: SOPTSharedPreference
+) : ViewModel() {
     val userId = MutableLiveData("")
     val userPw = MutableLiveData("")
+    private val _isSaveId = MutableLiveData(false)
```

Comment:
```suggestion
    private val _isSavedId = MutableLiveData(false)
```
혹은 `_isIdLoadedWithAutoLogin` 도 괜찮아보입니다

Reply:
`isIdLoadedAuto` 로 바꿨습니다~ 센스 대박이시네요 선생님👍